packages/lesswrong/server/migrations/20250503T040404.add_llmEvaluations_and_index.ts

```diff
@@ -0,0 +1,15 @@
+import AutomatedContentEvaluations from "../collections/automatedContentEvaluations/collection";
+import { addField, dropField, updateIndexes } from "./meta/utils";
+
+export const up = async ({db}: MigrationContext) => {
+  await addField(db, AutomatedContentEvaluations, "aiChoice");
+  await addField(db, AutomatedContentEvaluations, "aiReasoning");
+  await addField(db, AutomatedContentEvaluations, "aiCoT");
```

Comment:
I don't think this will work if these are all non-nullable fields without default values.

Reply:
Yeah, that's my top level comment on this PR